### ISSUE TITLE
Likelihood reconstruction analysis with lst-chain

### ISSUE THREAD

**Issue by calispac** [cta-observatory/cta-lstchain#449]
Hi all,

I am writing to inform you that I would like to perform a likelihood based analysis with the LST MC up to the sensitivity. 
The method is presented here : https://indico.cta-observatory.org/event/2558/contributions/22811/attachments/16915/22263/eas_images_fitting.pdf
I have obtained comparison (between a standard reconstruction and this method) for the SST-1M Monte Carlo simulation. Since the LST MC has been already processed by many people it is a good opportunity for me to compare the sensitivities. 

The method itself should make the DL0->DL1 transition without passing by reduced waveforms. However it uses a fit to which an initialization has to be used. I will use the "standard" reconstruction of lst-chain has an initialization. 
The end parameters are, in principle, the same to the ones in the DL1 container. So for the DL1---> sensitivity I will use lst-chain.

I have already created a fork of this repo. on my github personal page and I will commit progress onto it.

Best regards,
Cyril

**Comment by rlopezcoto** [comment]
Hi @calispac, thank you for letting us know. You may want to have a look to the work presented by @thomasgas in the LST Bootcamp: https://indico.cta-observatory.org/event/1987/timetable/#20181127.detailed regarding LST Analysis reconstruction with ImPACT

**Comment by calispac** [comment]
That's good to know! Thanks. I have also implemented some classes to handled the fitting part. I will adapt mine to resemble the ImpactReconstructor

**Comment by calispac** [comment]
Inheriting from `ctapipe.reco.Reconstrucor`: https://cta-observatory.github.io/ctapipe/_modules/ctapipe/reco/reco_algorithms.html seems a good start.

**Comment by calispac** [comment]
Hello again,

I have trouble to find the sim_telarray configuration files associated to the DL0 Monte-Carlo events that are in `/fefs/aswg/data/mc/DL0/20190415/gamma/south_pointing`. In particular, I am interested at the :

- single photo-electron spectra
- the pulse shapes (HG and LG)
- the ADC to p.e. amplitude ratio (HG and LG)
Thanks in advance !

**Comment by calispac** [comment]
I went to dig into the simtel files with `eventio`. I found a reference pulse shape (actually two). 
This is what I have found :+1: 
![image](https://user-images.githubusercontent.com/23462345/86918889-93749d80-c127-11ea-81b8-4122f82afbb1.png)

Is it normal that the two channels have the same reference pulse shape ? I mean was it wanted to simulated both channels with the same template ?

**Comment by moralejo** [comment]
> I went to dig into the simtel files with `eventio`. I found a reference pulse shape (actually two).
> This is what I have found :
> ![image](https://user-images.githubusercontent.com/23462345/86919742-00d4fe00-c129-11ea-8281-c8e9fd46df2f.png)
> 
> Is it normal that the two channels have the same reference pulse shape ? I mean was it wanted to simulated both channels with the same template ?

Yes, this is one of the obsolete things in Prod3 simulations. We have now different shapes for high and low-gain single pe pulses.

**Comment by calispac** [comment]
Thank you for the confirmation. I have found the configuration I needed for Prod3.
